packages/terra-menu/src/MenuItem.scss

```diff
@@ -10,15 +11,29 @@
       &:focus {
         background-color: var(--terra-menu-item-hover-color, #ebf6fd);
       }
-    } 
+    }
+  }
+
+  .item:active:not(.is-disabled),
+  .active {
+    background-color: var(--terra-menu-item-active-color, #ebf6fd);
+
+    .title {
+      color: var(--terra-menu-item-active-text-color, #1c1f21);
+    }
   }
 
   .is-disabled {
     outline: 0;
     
     .title {
+      color: var(--terra-menu-item-disabled-text-color, #1c1f21);
```

Comment:
Any objections to renaming this? 

`--terra-menu-item-disabled-*title*-color`

Reply:
I would gather additional opinions before making a decision but because this is a new variable I would vote to name it --terra-menu-item-disabled-**title**-color or to remap the the css declaration of `.title` to be `.text`

Reply:
It looks like the prop name is `text` so I would lean towards the second approach